juntagrico/templates/depot.html

```diff
@@ -45,7 +45,7 @@ <h3>
                 {% trans "Beschreibung" %}
             </div>
             <div class="col-md-9">
-                {{ depot.description|linebreaksbr|urlize }}
+                {{ depot.description|linebreaksbr|urlize|safe }}
```

Comment:
`urlize` only works on plain text: See note here https://docs.djangoproject.com/en/3.1/ref/templates/builtins/#urlize
Also if applied before `safe` is breaks the html.
`linebreaksbr` before `safe` also breaks the html. Applying it afterwards adds extra newlines.

We might have to create a filter that checks, if the content is HTML already and then applies the filters accordingly.

Reply:
i created a filter which checks if the richtexteditor is enabled or not and if not does the urlize and linebreacksbr